elementary/monitor/api/alerts.py

```diff
@@ -15,9 +15,10 @@
 
 
 class AlertsAPI(APIClient):
-    def __init__(self, dbt_runner: DbtRunner, elementary_database_and_schema: str):
+    def __init__(self, dbt_runner: DbtRunner, elementary_database_and_schema: str, config):
```

Comment:
```suggestion
    def __init__(self, dbt_runner: DbtRunner, config: Config, elementary_database_and_schema: str):
```
Not that important but the `config` is of higher priority and I think it should be before `elementary_database_and_schema`.